packages/@glimmer/debug/lib/render/fragment.ts

```diff
@@ -30,31 +30,31 @@ export class Fragment<T extends FragmentType = FragmentType> {
   static integer(
     this: void,
     value: number,
-    options?: Omit<IntegerFragment, 'kind' | 'value'> | undefined
+    options?: Omit<IntegerFragment, 'kind' | 'value'>
```

Comment:
You don't need `| undefined` in parameter positions, and this (and many other cases like it) were
caught by the new lint rules.